src/Engine/ProtoCore/Lang/BuiltInFunctionEndPoint.cs

```diff
@@ -1731,7 +1734,7 @@ internal static StackValue Remove(StackValue sv1, StackValue sv2, ProtoCore.DSAS
 
             var svArray = runtimeCore.Heap.ToHeapObject<DSArray>(sv1).Values.ToArray();
             int length = svArray.Length;
-            int indexToBeRemoved = (int)sv2.opdata;
+            int indexToBeRemoved = (int)sv2.IntegerValue;
```

Comment:
That was what I did, but later realize that we should support `Int64`. `int` in DS is 64 bits, casting to 32 bit `int` is only in DS engine.